### PR TITLE
Typography improvement to Thank You page

### PR DESCRIPTION
**Pull request by eremitik**
Hi! This is my first PR to an open-source project and wanted to start small. Hope this is acceptable and I can contribute more in the near future.

### CODE REVIEW

File: pages/thank-you.js [hunk 1]
```diff
@@ -4,34 +4,33 @@ export default function Home() {
...
-              <p>{'With gratitude from Margarita Humanitarian Foundation.'}</p>
+              <h1 className="mb-5 text-5xl font-bold">{'Thank You!'}</h1>
               <p>
                 {
-                  'Thanks so much for your generosity. We promise to use it to help the causes you indicated your support for, and to be extra thoughtful about putting it to good use.'
+                  'With gratitude from Margarita Humanitarian Foundation. Thanks so much for your generosity. We promise to use it to help the causes you indicated your support for, and to be extra thoughtful about putting it to good use.'
                 }
               </p>
-              <h2 className="m-5 text-3xl font-bold">{'Receipt to Follow'}</h2>
+              <h2 className="m-0 mt-10 mb-2 text-3xl font-bold">{'Receipt to Follow'}</h2>
```
Comment (RedFox0x20) [inline pages/thank-you.js:19]: Please put a newline before the `{` and after the `}`
Alternatively run `eslint . --fix` if you have eslint installed.

File: pages/thank-you.js [hunk 2]
```diff
@@ -46,16 +45,16 @@ export default function Home() {
                   ". We're looking for volunteer contributors to help make it better. Report bugs or propose ideas for improvements by submitting GitHub issues, or even submit little pull requests as proposals."
                 }
               </p>
-              <h3 className="m-5 text-xl font-bold">{'Reach Out to Us'}</h3>
+              <h3 className="m-0 mt-10 mb-2 text-3xl font-bold">{'Reach Out to Us'}</h3>
```
Comment (RedFox0x20) [inline pages/thank-you.js:48]: Please put a newline before the `{` and after the `}`
Alternatively run `eslint . --fix` if you have eslint installed.

**Review by RedFox0x20 — CHANGES_REQUESTED** [review]
Hello @eremitik welcome to open source! This was also the first project that I contributed to too!

You've made some nice stylistic changes to the page, it looks much tidier and you have likely improved the pages readability a bit!

You'll see that your PR failed to deploy with Netlify, this is due to some linting issues, thankfully these are easy to fix! Please see the comments I have left on the two problematic lines!

**Comment by audreyfeldroy** [comment]
Hey @eremitik! You've done a great job with this PR so far and we want to help you get it merged. 

Let us know if you get stuck on adding a commit to address the above...we're happy to help you through any obstacles right here, or even [over chat](https://app.element.io/#/room/#mhf:matrix.org) or screenshare.

File: pages/thank-you.js (hunk 1 above)
Comment (eremitik) [reply pages/thank-you.js]: Thank you very much for the detailed comments, I have made the changes as suggested!

**Comment by audreyfeldroy** [comment]
Merged, excellent work @eremitik! Congrats on your first open-source PR. You did it perfectly and we appreciate your contribution very much.

**Pull request merged**